src/Discord/Parts/User/Member.php

```diff
@@ -402,7 +402,7 @@ protected function getJoinedAtAttribute(): ?Carbon
     public function getAvatarAttribute(string $format = 'jpg', int $size = 1024): string
```

Comment:
```suggestion
    public function getAvatarAttribute(string $format = 'jpg', int $size = 1024): ?string
```

Should change the type hint instead